src/docfx/cli/Program.cs

```diff
@@ -67,20 +73,26 @@ private static (string command, string docset, CommandLineOptions options) Parse
             var docset = ".";
             var options = new CommandLineOptions();
 
+            if (args.Length == 0)
+            {
+                // Show usage when just running `docfx`
+                args = new [] { "--help" };
+            }
+
             ArgumentSyntax.Parse(args, syntax =>
             {
                 // Restore command
                 // usage: docfx restore [docset]
-                syntax.DefineCommand("restore", ref command, "restores dependencies before build");
-                syntax.DefineParameter("docset", ref docset, "docset path that contains docfx.yml");
+                syntax.DefineCommand("restore", ref command, "Restores dependencies before build.");
+                syntax.DefineParameter("docset", ref docset, "Docset directory that contains docfx.yml.");
 
                 // Build command
-                // usage: docfx build [docset] [-o/--out output] [-l/--log log] [--legacy]
-                syntax.DefineCommand("build", ref command, "builds a folder containing docfx.yml");
-                syntax.DefineOption("o|out", ref options.Output, "output folder");
-                syntax.DefineOption("l|log", ref options.Log, "path to log file");
```

Comment:
`l|log`?

Reply:
`log` is short enough, I'd like to reserve `l` for something else